cmd/notation/cert/list.go

```diff
@@ -22,6 +22,20 @@ func certListCommand(opts *certListOpts) *cobra.Command {
 		Use:     "list",
 		Aliases: []string{"ls"},
 		Short:   "List certificates in the trust store.",
+		Long: `List certificates in trust store
+
+Example - List all certificate files stored in the trust store
+  notation certificate ls
+
+Example - List all certificate files of trust store "acme-rockets"
+  notation cert ls --store "acme-rockets"
+
+Example - List all certificate files from trust store of type "ca"
+  notation cert ls --type ca
+
+Example - List all certificate files from trust store "wabbit-networks" of type "signingAuthority"
+  notation cert ls --type signingAuthority --store "wabbit-networks"
```

Comment:
Same question. I cannot find type of "signingAuthority" in the spec https://github.com/notaryproject/notaryproject/blob/main/specs/trust-store-trust-policy.md